### DISCUSSION TITLE
EMQX open source  docker installation problem in windows server 2022

### DISCUSSION THREAD

**Discussion by mnhpias** [emqx/emqx#14115] [Q&A]
I'm trying to install EMQX open source in windows server 2022. I've installed docker in windows, docker version is : `Docker version 27.3.1, build ce12230` .

When I had tried to install through windows powershell using : `docker pull emqx/emqx:5.8.1`
I got the output: `no matching manifest for windows/amd64 10.0.20348 in the manifest list entries`

Then I downloaded emqx-5.8.1-docker-amd64.tar.gz and tried  to install using : `docker load -i C:\Users\Administrator\Downloads\emqx-5.8.1-docker-amd64.tar.gz`
I got the output: `cannot load linux image on windows`

How can I insatll EMQX open source in windows server 2022?

**Comment by zhongwencool** [comment]
Maybe you should enable WSL 2 and Docker for Linux Containers on windows 2022

link: 
1. https://serverfault.com/questions/1114577/run-a-linux-docker-container-on-windows-server-2022
2. https://techcommunity.microsoft.com/t5/itops-talk-blog/wsl2-now-available-on-windows-server-2022/ba-p/3447570
